components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java

```diff
@@ -61,4 +62,10 @@ public interface ToolConfig {
      * @param targetDir path to the target directory
      */
     void setTargetDir(String targetDir);
+    
+    public String getSpecBasePath();
```

Comment:
```suggestion
   String getSpecBasePath();
```

Reply:
No need to have the public access modifier in the interface methods

Reply:
Add javadocs for the newly added interface methods

Reply:
fixed in 
ab4762c